src/wallet/rpczerocoin.cpp

```diff
@@ -739,8 +739,8 @@ UniValue spendzerocoinmints(const JSONRPCRequest& request)
     UniValue arrMints = params[0].get_array();
     if (arrMints.size() == 0)
         throw JSONRPCError(RPC_WALLET_ERROR, "No zerocoin selected");
-    if (arrMints.size() > 7)
-        throw JSONRPCError(RPC_WALLET_ERROR, "Too many mints included. Maximum zerocoins per spend: 7");
+    if (arrMints.size() > Params().Zerocoin_MaxSpendsPerTransaction())
+        throw JSONRPCError(RPC_WALLET_ERROR, "Too many mints included. Maximum zerocoins per spend: Params().Zerocoin_MaxSpendsPerTransaction()");
```

Comment:
You would print the string as:
`throw JSONRPCError(RPC_WALLET_ERROR, strprintf("Too many mints included. Maximum zerocoins per spend: %d", Params().Zerocoin_MaxSpendsPerTransaction());`

Reply:
Thanks Presstab!